string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+
+function reverseWord(word) {
```

Comment:
I think you can find a way to do this more succinctly. Remember that loops can go forwards and backwards!